File: SDKs/amazonaws.com/batch/2016-08-10/java/lib/src/main/java/openapisdk/models/shared/JqStatusEnum.java
package openapisdk.models.shared;


public enum JqStatusEnum {
    CREATING("CREATING"),
    UPDATING("UPDATING"),
    DELETING("DELETING"),
    DELETED("DELETED"),
    VALID("VALID"),
    INVALID("INVALID");

    public final String value;

    private JqStatusEnum(String value) {
        this.value = value;
    }
}

File: SDKs/amazonaws.com/batch/2016-08-10/python/src/sdk/models/shared/jqstatus_enum.py
import dataclasses
from enum import Enum

class JqStatusEnum(str, Enum):
    CREATING = "CREATING"
    UPDATING = "UPDATING"
    DELETING = "DELETING"
    DELETED = "DELETED"
    VALID = "VALID"
    INVALID = "INVALID"

